### ISSUE TITLE
Support Kubernetes 1.8.0

### ISSUE THREAD

**Issue by robhamk** [kubernetes/minikube#2020]
I use homebrew to install minikube which version is v0.22.2. The command is "brew cask install minikube"
My kubectl is the latest version 1.8.0. I installed it via "brew install kubectl"
Everything was going to fine but...
When I started to run minikube start --vm-driver=xhyve
Invalid Kubernetes version.
The following Kubernetes versions are available: 
	- v1.7.5
	- v1.7.4
	- v1.7.3
	- v1.7.2
	- v1.7.0
...
Does it mean minikube supports v.1.7.5 only?

**Comment by webwurst** [comment]
You can try different Kubernetes versions with the experimental `kubeadm` bootstrapper. The following works for me:
```
minikube start --kubernetes-version v1.7.7 kvm --bootstrapper kubeadm
```
But there is on open issue for running `v1.8.0` this way: https://github.com/kubernetes/minikube/issues/2018

**Comment by joan38** [comment]
Need `1.8.0` for https://github.com/kubernetes/kubernetes/issues/28918

**Comment by dlorenc** [comment]
This was merged in #2060. We'll get a release and build out soon.
